### DISCUSSION TITLE
关于Stash All Changes只能保存一个Stash的问题

### DISCUSSION THREAD

**Discussion by EasyGame-Vac** [community/community#54515] [Programming Help]
### Select Topic Area

Product Feedback

### Select Feature Area

- [ ] Issues
- [X] Projects

### Body

优化建议：在正常使用中，Stash All Changes只能保存一个，感觉有点怪，如果我此时已经有一个Changes的话，再次Stash会导致上个Changes直接被丢失，希望能够以堆栈的形式支持多个Changes。
BUG：在非正常使用中，比如我现在有一个Excel表被打开状态，Stash All Changes会出错，UI界面也没有刷新，此时我认为是没有Stash成功。但实际上是会创建一个Changes的，只是UI界面没有刷新。当多次Stash All Changes，虽然每次都会出错，但每次都会产生一个Changes，当关闭Excel后，重新选择该软件就会刷新界面，此时可以Restore多个Changes

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Hi @Vac-HongChen , thanks for participating!
We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.

**Comment by EasyGame-Vac** [comment]
Optimization suggestion: During normal usage, the "Stash All Changes" feature can only save one change, which feels a bit awkward. If I already have a change at the moment and I stash again, it will cause the previous change to be lost directly. It would be great to support multiple changes in a stack-like manner.

Bug: During abnormal usage, for example, when I have an Excel spreadsheet open, using "Stash All Changes" will result in an error and the UI interface doesn't refresh. In such cases, I mistakenly assume that the stash was unsuccessful. However, in reality, a change is created each time, but the UI interface doesn't refresh. When "Stash All Changes" is used multiple times, although errors occur each time, a change is generated each time. When the Excel is closed and the software is reopened, the interface refreshes, allowing the recovery of multiple changes.

**Comment by Hutch53** [comment]
@localhost=$Root:
